test/metabase/api/card_test.clj

```diff
@@ -364,7 +364,7 @@
                   Table     [{table-id :id}    {:db_id database-id, :name "CATEGORIES"}]
                   Card      [card              {:dataset_query {:database database-id
                                                                 :type     :native
-                                                                :native   {:query "SELECT COUNT(*) FROM CATEGORIES;"}}}]]
+                                                                :native   {:query "SELECT COUNT(*) FROM CATEGORIES"}}}]]
```

Comment:
if the driver doesn't support terminating semicolons?

Reply:
Correct, Drill (I think... it's possible it's Spark SQL) doesn't like them and they are not necessary.